### ISSUE TITLE
[BITV] 9.4.1.2/3.7 - The "Actions" menu buttons in the table should be implemented as buttons and not as links. (2)

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/server#37104]
- [ ] Fix
- [ ] Backport

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-bfbf07e5c3-v3-n7

</details>

- [ ] 9.4.1.2/10.7 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-bfbf07e5c3-v10-n7

**Comment by Fenn-CS** [comment]
Parking this issue while waiting for the full migration of the files app to vuejs

**Comment by skjnldsv** [comment]
Done with F2V
